oms-azurestorageanalytics-solution/azureStorageAnalytics.json

```diff
@@ -0,0 +1,2497 @@
+                        "yAxis": {
+                          "isLogarithmic": false,
+                          "units": {
+                            "baseUnitType": "",
+                            "baseUnit": "",
+                            "displayUnit": ""
+                          },
+                          "customLabel": ""
+                        }
+                      }
+                    }
+                  ]
+                }
+              },
+              {
+                "Id": "SingleQueryDonutBuilderBladeV1",
+                "Type": "Blade",
+                "Version": 0,
+                "Configuration": {
+                  "General": {
+                    "title": "TABLE - TOTAL REQUESTS ",
+                    "newGroup": false,
+                    "icon": "",
+                    "useIcon": false
+                  },
+                  "Header": {
+                    "Title": "Total Request per Storage Account",
+                    "Subtitle": ""
+                  },
+                  "Donut": {
+                    "Query": "Type=AzureStorage_CL     MetricName_s=MetricsTransactions  StorageService_s=Table  TimeGenerated>NOW-1HOUR |measure sum(TotalRequests_d) by StorageAccount_s",
+                    "CenterLegend": {
+                      "Text": "Total",
+                      "Operation": "Sum",
+                      "ArcsToSelect": []
+                    },
+                    "Options": {
+                      "colors": [
+                        "#00188f",
+                        "#0072c6",
+                        "#00bcf2"
+                      ],
+                      "valueColorMapping": []
+                    }
+                  },
+                  "List": {
+                    "Query": "Type=AzureStorage_CL     MetricName_s=MetricsTransactions  StorageService_s=Table  TimeGenerated>NOW-1HOUR |measure sum(TotalRequests_d) by StorageAccount_s",
+                    "HideGraph": false,
+                    "enableSparklines": false,
+                    "operation": "Summary",
+                    "ColumnsTitle": {
+                      "Name": "Storage Account",
+                      "Value": "RequestCount"
+                    },
+                    "Color": "#0072c6",
+                    "thresholds": {
+                      "isEnabled": false,
+                      "values": [
+                        {
+                          "name": "Normal",
+                          "threshold": "Default",
+                          "color": "#009e49",
+                          "isDefault": true
+                        },
+                        {
+                          "name": "Warning",
+                          "threshold": "60",
+                          "color": "#fcd116",
+                          "
```

Comment:
Why  should we  remove this  one, Its  defines the indicators next to lists in OMS designer?
.by the way I have removed azurestorageanalytics.json and upload is as azuredeploy.json as  azurequickstart guide.